packages/components/src/RatioBar/RatioBar.component.js

```diff
@@ -31,7 +31,7 @@ function getEmptyValues(amount, total) {
 	return { percentage: (1 - amount / total) * 100, amount: total - amount };
 }
 
-function getLabel(amount, total) {
+function getLabel(amount, errors, total) {
```

Comment:
why do you need to change the fn? just pass the amount already computed with errors

Reply:
`null + 0` returns `0` and the N/A case no longer works

Reply:
UTs saved me 🙃